packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -480,6 +515,21 @@ QUnit.module('interaction via swipe', {
         });
     });
 
+    QUnit.test('when only one item is visible, swipe action does not move the current visible item', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false }
+            ]
+        });
+        const $itemContainer = $multiView.find(toSelector(MULTIVIEW_ITEM_CONTAINER_CLASS));
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(-0.1);
+        assert.strictEqual(position($itemContainer), 0, 'container did not move');
+        pointer.swipeEnd();
+    });
+
```

Comment:
[test coverage]
Please add tests for scenarios:
- swipe to right should skip hidden items | [initial selectedIndex=0, expected 2]
- swipe to left should skip hidden items | [initial selectedIndex=2, expected 0]
U have similar tests for `loop` but not for a default behavior

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)